lib/bracket-matcher.coffee

```diff
@@ -24,6 +24,7 @@ class BracketMatcher
     "‹": "›"
 
   toggleQuotes: (includeSmartQuotes) ->
+    return if @pairedCharacters? and includeSmartQuotes is (@pairedCharacters isnt @defaultPairs)
```

Comment:
What do you think about clarifying this `if` statement? At a first glance, it's not entirely clear to me what are we checking here. Alternatively, can we get away with not adding this guard clause at all? :thought_balloon: